src/store/actions.ts

```diff
@@ -164,7 +164,6 @@ export const resetAnalyzeName = ({ commit, getters }): void => {
   if (!getters.getUserCancelledAnalysis) {
     commit('mutateAnalysisJSON', null)
   }
-  commit('mutateCorpNum', '')
```

Comment:
After doing lots of digging, I found out that every time, the `corpNum` is being set as an empty string here. There was nothing to counter this for Alterations, Restorations, and Name Change.

So, I moved this line to the else statement below.

Reply:
Sev, please check latest commit.

Here's my thought process: The `corpNum` is being manually set in the code in a couple of places. When it's been manually set (like in REH for example), I don't want to run mutate the `corpNum` to being an empty string.

Reply:
This "feels" better. It's clearing the value in the same place as the others, and there's similar code to yours just above. So it's probably OK.

If we had time, it would be nice to understand why `resetAnalyzeName` is being called after we set a value. But we don't, so if this works then I'm fine with it.

Reply:
How about adding a comment that we set corpNum earlier in the path and we don't want to reset it here while analyzing the name?

(Now it "feels" like corpNum has nothing to do with name analysis, so why is it being cleared here at all? Sigh.)

Reply:
You mean add a comment in the code explaining why is the if statement there Sev and why are we setting the corpNum as an empty string? I didn't quite get that.

Reply:
Yes, why is this here / needed? But this may all be obsolete if you remove corpNum from the name analysis flow (since it's not at all related, I think).

The root cause is that we previously did name analysis and then prompted for the corpNum, but now we look up the corpNum and then do the name analysis.

Reply:
Yes, agreed. This line was written 2 years ago. I think it is indeed obsolete. Going to take a deeper dive and make sure if it's obsolete, I can remove the line all together.

Reply:
I've confirmed Sev. This line is indeed obsolete.

I've tried going through the process of creating an NR through various flows after removing this line. The corpNum is being properly set when REH is selected for example while it's empty in a NEW flow.

Reply:
Thanks for testing. Are the other 2 flows OK? And so this PR is not needed?

Reply:
Awesome. This conversation is fine by me (ok to resolve).

Reply:
Why don't you merge this fix and open another PR for other fixes?

Reply:
Sounds good. I'm just waiting for another approval!